context/context_test.go

```diff
@@ -54,6 +54,41 @@ func TestLoadFlatContextFromFile(t *testing.T) {
 	}
 }
 
+func TestLoadContextWithArgs(t *testing.T) {
+	ctx, err := LoadContext("testdata/flat-with-args-test.yaml", &noExplicitVars)
+
+	if err != nil {
+		t.Error(err)
+		t.Fail()
+	}
+
+	expected := Context{
+		Name: "k8s.prod.mydomain.com",
+		ResourceSets: []ResourceSet{
+			{
+				Name:   "some-api",
+				Path:   "some-api",
+				Values: make(map[string]interface{}, 0),
+				Args: []string{
+					"--as=some-user",
+					"--as-group=hello:world",
+					"--as-banana true",
```

Comment:
Is this particular example actually a valid flag? I would expect that the space separation without a `=` token here means that it must be passed as `["--as-banana", "true"]`.

Reply:
I'm pretty sure you're right! I've changed it the test and example in c8c894d